src/main/java/org/broadinstitute/hellbender/tools/walkers/RevertBaseQualityScores.java

```diff
@@ -23,7 +23,7 @@
         programGroup = ReadProgramGroup.class
 )
 
-public class RevertBaseQualityScores extends ReadWalker {
+public final class RevertBaseQualityScores extends ReadWalker {
```

Comment:
yes, unrelated but i'm following the [boy scout rule] (http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) here